includes/i18n.php

```diff
@@ -143,7 +143,7 @@
         'paypal-email-address-is-required'   => _fs_text( 'PayPal email address is required.' ),
         'processing'                         => _fs_text( 'Processing...' ),
         'non-expiring'                       => _fs_text( 'Non-expiring' ),
-        'account-is-pending-activation'      => _fs_text( 'Account is pending activation.' ),
+        'account-is-pending-activation'      => _fs_text( 'Account is pending activation. Please check your email and click the link to activate your account and then submit the affiliate form again.' ),

```

Comment:
@swashata you can revert all the changes in the `i18n.php` file as it is no longer in use. Our previous approach was to have all the translations in the same file within the same array. But it was taking unnecessary RAM so we transitioned to "inline" translations. We kept it for backward compatibility. Now that it's been years already, it's safe to get rid of that file and usage. I'll create a task for it in Asana.